docs/HowTo/Interact/APIs/Using-JSON-RPC-API.md

```diff
@@ -2,12 +2,19 @@
 description: How to access the Hyperledger Besu API using JSON-RPC
 ---
 
-# JSON-RPC over HTTP and WebSockets
+# JSON-RPC over HTTP, WebSockets and IPC
 
 To enable JSON-RPC over HTTP or WebSockets, use the
 [`--rpc-http-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-http-enabled) and
 [`--rpc-ws-enabled`](../../../Reference/CLI/CLI-Syntax.md#rpc-ws-enabled) options.
 
+To enable JSON-RPC over an IPC socket, use the
```

Comment:
We should perhaps mention the default path here? or link to the "Socket path" topic?

Reply:
Added link